### PR TITLE
Nroe/python/display cards

### PR DESCRIPTION
**Pull request by nattyroe**
Updates Cards when refresh is called. Also updates Mrs White to Dr Orchid

### CODE REVIEW

File: backend-server/collections/suspects.js [hunk 1]
```diff
@@ -21,8 +21,8 @@ exports.suspects = [
     startingLocation: locations.find(location => location.name === 'Conservatory-Library-Hallway'),
     color: 'blue'
   }, {
-    name: 'Mrs. White',
+    name: 'Dr. Orchid',
```
Comment (DRSisco) [inline backend-server/collections/suspects.js:24]: I didn't know they made this change. I wonder why? To be more inclusive?

File: backend-server/models/game.js [hunk 2]
```diff
@@ -130,14 +130,14 @@ const Game = class Game{
...
       this.winner = guessingPlayer;
       this.broadcast(`${guessingPlayer.username} has won!`);
-      return;
+      return true;
     }
 
     guessingPlayer.failed = true
     if (this.players.some(player => player.failed === false)) {
-      this.turn.nextPlayer()
+      this.turn.endTurn()
```
Comment (DRSisco) [inline backend-server/models/game.js:138]: Good catch!

File: backend-server/handlers/gameplay/accusation.js [hunk 3]
```diff
@@ -6,9 +6,9 @@ module.exports = (req, res) => {
   if (parameterCheck(req, requiredParams)) {
     try {
       let game = Game.find(req.body.gameId)
-      game.makeAccusation(req.body.username, req.body.accusation)
-      res.json('')
-    } catch(e) {
+      result = game.makeAccusation(req.body.username, req.body.accusation)
+      res.json(result)
```
Comment (DRSisco) [inline backend-server/handlers/gameplay/accusation.js:10]: :+1:

**Review by DRSisco — APPROVED** [review]
Looks good and thanks for making the changes you needed in the backend.

File: backend-server/collections/suspects.js (hunk 1 above)
Comment (nattyroe) [reply backend-server/collections/suspects.js:24]: I think it was a refresh to publish a new version, like how Monopoly is updating their community chest cards. But she is the one that is in the version my parents have so I updated to match the cards.

**Pull request merged**